widgetssdk/src/main/java/com/glia/widgets/chat/adapter/UploadAttachmentAdapter.kt

```diff
@@ -181,13 +183,13 @@ class ViewHolder(
     ) {
         removeItemButton.contentDescription = stringProvider.getRemoteString(
             R.string.chat_upload_remove_accessibility_label,
-            displayName
+            StringKeyPair(StringKey.NAME, displayName)
         )
         itemView.contentDescription = stringProvider.getRemoteString(
             R.string.android_chat_file_accessibility,
-            displayName,
-            size,
-            getStatusIndicatorText(attachment.attachmentStatus)
+            StringKeyPair(StringKey.NAME, displayName),
+            StringKeyPair(StringKey.SIZE, size ?: "N/A"),
```

Comment:
@Pelkar Does it make sense to move "N/A" to strings (at least, non-translatable)?

Reply:
There needs to be a value here, because if we otherwise size: {NOTHING} is weird, N/A should be the closest to multilanguage placeholder to this as we can manage. Otherwise we should start using different string res